### PR TITLE
update(OWNERS): remove nestorsalceda

### PR DESCRIPTION
**Pull request by jasondellaluce**
**What type of PR is this?**

/kind documentation

**Any specific area of the project related to this PR?**

**What this PR does / why we need it**:

This supersedes #373 due to it being not mergeable. See https://github.com/falcosecurity/falco/issues/2143#issuecomment-1195195353.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

/cc @falcosecurity/charts-maintainers

### CODE REVIEW

File: falco/OWNERS [hunk 1]
```diff
@@ -1,3 +1,2 @@
 emeritus_approvers:
   - bencer
-  - nestorsalceda
```
Comment (cpanato) [inline falco/OWNERS:3]: this is ok to remove?

Comment (jasondellaluce) [reply falco/OWNERS:3]: I replicated the same changes of #373 from @nestorsalceda due to https://github.com/falcosecurity/falco/issues/2143#issuecomment-1195195353.

From https://github.com/falcosecurity/falco/issues/2143, he asks to be removed from everywhere. The linked PRs also  involve `emeritus_approvers` without providing specific rationale for that section.

Comment (cpanato) [reply falco/OWNERS:3]: got it, ok 🤔

**Review by cpanato — APPROVED** [review]
/approve
/lgtm

thanks for all nestor!

**Pull request merged**